Dockerfile

```diff
@@ -511,9 +511,17 @@ RUN useradd --create-home --gid docker unprivilegeduser \
  && chown -R unprivilegeduser /home/unprivilegeduser
 # Let us use a .bashrc file
 RUN ln -sfv /go/src/github.com/docker/docker/.bashrc ~/.bashrc
-# Activate bash completion and include Docker's completion if mounted with DOCKER_BASH_COMPLETION_PATH
+# Activate bash completion
 RUN echo "source /usr/share/bash-completion/bash_completion" >> /etc/bash.bashrc
-RUN ln -s /usr/local/completion/bash/docker /etc/bash_completion.d/docker
+# Include Docker's completion if mounted with DOCKER_BASH_COMPLETION_PATH
```

Comment:
Somewhat wondering if we still need the `DOCKER_BASH_COMPLETION_PATH ` code 🤔 It was added in https://github.com/moby/moby/pull/33801 to help with the manually written bash-completion scripts, so ... perhaps? That said, we also have the `moby-bin` image, so maybe we could create something in the docker/cli repository to help spin up a docker-in-docker environment with the latest daemon and the current CLI (and completion script)

Reply:
Yeah looks like we could remove it from moby and do that in the CLI which would be more appropriate place for shell completion development.